### ISSUE TITLE
Ensure report.package_version is increased on every new release

### ISSUE THREAD

**Issue by epou** [Mosquito-Alert/Mosquito-Alert-Mobile-App#36]
We need to ensure that on every new app release, the report.package_version in `lib/utils/Utils.dart` is increased to be able to detect/correct buggy behaviors on the server side.

I've seen that `report.package_version` was hardcoded to '32' in: [e11ce6af9fa699338297b6375f2170916bcbc1be](https://github.com/Mosquito-Alert/Mosquito-Alert-Mobile-App/commit/e11ce6af9fa699338297b6375f2170916bcbc1be#diff-d563b7507e992f3dd34068ad60d6eeb363e38baf1021cc42e7c57afd3eb14b11R84)

A possible solution could be using the `package_version` library again to be consistent with the `version` field set in the `package.yaml` file.

NOTE: [package_version](https://pub.dev/documentation/package_info/latest/) seems to have been discontinued since 2021 and replaced with [package_version_plus](https://pub.dev/packages/package_info_plus)

**Comment by pablogalve** [comment]
> A possible solution could be using the package_version library again to be consistent with the version field set in the package.yaml file.

Yes, I agree. That's what I meant in my previous message. Maybe my communication was not clear:

> If the second, we could consider instead to use the app version for consistency (currently 1709)

The other points:
> Just to keep in mind that the backend accepts Integers only

> We need to ensure that on every new app release, the report.package_version in lib/utils/Utils.dart is increased to be able to detect/correct buggy behaviors on the server side.

I'd say that using the buildNumber kills those 2 birds in one shot, since 1709, 1710, ... are integers and Google Play forces this number to be increased (it rejects any update with same version), so it simplifies debugging on server side by having the build number rather than an unrelated number
So yeah, I vote +1 for your proposal of using [package_version_plus](https://pub.dev/packages/package_info_plus)

**Comment by epou** [comment]
Looks good to me, the only thing is that there would be a huge gap between the current `report.package_version` used in the DB (which is 32) and the following version -> 1709.

@aescobarr , @JohnPalmer  what are your thoughts on this?

**Comment by aescobarr** [comment]
As long as package.versions are unique and are consistently associated with different release versions, I am ok with any solution.
